extensions/azurePublish/src/node/deploy.ts

```diff
@@ -159,6 +170,50 @@ export class BotProjectDeploy {
     }
   }
 
+  /**
+   * update the skill related settings to skills' manifest
+   * @param hostname hostname of web app
+   * @param msAppId microsoft app id
+   * @param skillSettingsPath the path of skills manifest settings
+   */
+  private async updateSkillSettings(
+    profileName: string,
+    hostname: string,
+    msAppId: string,
+    skillSettingsPath: string,
+    settings: any
```

Comment:
We have a type for Bot Project Settings on the server

Reply:
use `DialogSetting` type